tests/webpack.js

```diff
@@ -0,0 +1,35 @@
+var test = require('tape');
+var webpack = require('webpack');
+
+var conf = {
+    entry: {
+        index: ".."
+    },
+    output: {
+        path: "./output/",
+        filename: "bundle.js"
+    },
+    module: {
+        loaders: [{
+            test: /\.json$/,
+            loader: "json"
+        }]
+    },
+    // because client side doesn't have fs :)
+    node: {
+        fs: "empty"
+    }
```

Comment:
This is jsut a workaround for missbehaving packages.

You better should add
``` js
"browser": {
  "fs": false
}
```

to the package.json.